project/Dependencies.scala

```diff
@@ -2,8 +2,8 @@ import sbt._, Keys._
 
 object Dependencies {
 
-  val AkkaVersion = "2.4.14"
-  val AkkaHttpVersion = "10.0.0"
+  val AkkaVersion = "2.4.18"
+  val AkkaHttpVersion = "10.0.6"
```

Comment:
hm note to self that we should update this one right away when we have important releases of http,
will add a note to our releasing doc there